DEVELOPMENT.md

```diff
@@ -26,108 +26,37 @@ At this point you're waiting on us. We may suggest some changes, improvements or
-    #      - $JENKINS_PLUGIN/conf.yaml:/etc/datadog-agent/conf.d/jenkins.d/conf.yaml
-
-    ```
-1. If you wish to submit log using the Datadog Agent, you will have to configure the Datadog Agent properly by creating a `conf.yaml` file with the following content.
-
-    ```
-    logs:
-      - type: tcp
-        port: 10518
-        service: "jenkins"
-        source: "jenkins"
-    ```
 1. Set the `JENKINS_PLUGIN` environment variable to point to the directory where this repository is cloned/forked.
 1. Set the `JENKINS_PLUGIN_DATADOG_API_KEY` environment variable with your api key.
-1. Run `docker-compose -f <DOCKER_COMPOSE_FILE_PATH> up`.
-    - NOTE: This spins up the Jenkins docker image and auto mount the target folder of this repository (the location where the binary is built)
-    - NOTE: To see code updates, after re building the provider with `mvn clean package` on your local machine, run `docker-compose down` and spin this up again.
-1. Check your terminal and look for the admin password:
-    ```
-    jenkins_1    | *************************************************************
-    jenkins_1    | *************************************************************
-    jenkins_1    | *************************************************************
-    jenkins_1    |
-    jenkins_1    | Jenkins initial setup is required. An admin user has been created and a password generated.
-    jenkins_1    | Please use the following password to proceed to installation:
-    jenkins_1    |
-    jenkins_1    | <JENKINS_ADMIN_PASSWORD>
-    jenkins_1    |
-    jenkins_1    | This may also be found at: /var/jenkins_home/secrets/initialAdminPassword
-    jenkins_1    |
-    jenkins_1    | *************************************************************
-    jenkins_1    | *************************************************************
-    jenkins_1    | *************************************************************
-    ```
-
-1. Access your Jenkins instance http://localhost:8080
-1. Enter the administrator password in the Getting Started form.
-1. On the next page, click on the "Select plugins to be installed" unless you want to install all suggested plugins.
-1. Select desired plugins depending on your needs. You can always add plugins later.
-1. Create a user so that you don't have to use the admin credential again (optional).
-1. Continue until the end of the setup process and log back in.
-1. Go to http://localhost:8080/configure to configure the "Datadog Plugin", set your `API Key`.
-  - Click on the "Test Key" to make sure your key is valid.
+1. Optionally set the `GITHUB_SSH_KEY` and `GITHUB_SSH_KEY_PASSPHRASE` environment variables with the key and passphrase that can be used to access GitHub. This allows to automatically create GitHub credentials in Jenkins.   
+1. Run `mvn clean package` and `docker-compose -p datadog-jenkins-plugin -f docker/docker-compose.yaml up` from the directory where this repository is cloned/forked.
```

Comment:
```suggestion
1. Run `mvn clean package -DskipTests` and `docker-compose -p datadog-jenkins-plugin -f docker/docker-compose.yaml up` from the directory where this repository is cloned/forked.
```

Maybe? In this case you just want to run the dev environment.

On the other hand, I've executed the command `mvn clean package` but it gets stuck in the `InjectionTest`

```
[INFO]
[INFO] --- maven-surefire-plugin:3.0.0-M7:test (default-test) @ datadog ---
[INFO] Using auto detected provider org.apache.maven.surefire.junit4.JUnit4Provider
[INFO]
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running InjectedTest
```

Reply:
Yeah, I don't think it actually gets stuck (for me it takes a few minutes to complete that test), but certainly running the tests is not a prerequisite for spinning up a local Jenkins instance.